pkgs/by-name/ri/rimgo/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitea,
+  buildGoModule,
+  tailwindcss,
+}:
+buildGoModule rec {
+  pname = "rimgo";
+  version = "1.2.0";
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "rimgo";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-C878ABs978viVtIuv3fPn2F2anOg2GB/+f5jaCO13tc=";
+  };
+  preBuild = ''
+    ${tailwindcss}/bin/tailwindcss -i static/tailwind.css -o static/app.css -m
+  '';
+  vendorHash = "sha256-u5N7aI9RIQ3EmiyHv0qhMcKkvmpp+5G7xbzdQcbhybs=";
+  meta = with lib; {
+    description = "An alternative frontend for Imgur";
+    platforms = platforms.unix;
+    homepage = "https://codeberg.org/rimgo/rimgo";
+    license = with licenses; [agpl3Only];
+    maintainers = with maintainers; [quantenzitrone];
```

Comment:
```suggestion
    license = licenses.agpl3Only;
    maintainers = with maintainers; [ quantenzitrone ];
```

Reply:
i format with alejandra until RFC 101 gets merged, which will also reformat all of nixpkgs to the specified style
until then, any bikeshedding about meaningless style attributes in PRs is useless